doc/lua_api.txt

```diff
@@ -3110,8 +3110,10 @@ An `InvRef` is a reference to an inventory.
 * `add_item(listname, stack)`: add item somewhere in list, returns leftover `ItemStack`
 * `room_for_item(listname, stack):` returns `true` if the stack of items
   can be fully added to the list
-* `contains_item(listname, stack)`: returns `true` if the stack of items
-  can be fully taken from the list
+* `contains_item(listname, stack, [match_meta])`: returns `true` if
+  the stack of items can be fully taken from the list.
+  If `match_meta` is false, only the items' names are compared.
+  The default is false.
```

Comment:
Clearer and shorter to have the statement of default in the same sentence as 'match meta'
` If `match_meta` is false, only the items' names are compared (default:false).`